misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -8,18 +9,19 @@ import misk.exceptions.UnauthenticatedException
 import misk.exceptions.UnauthorizedException
 import misk.scope.ActionScoped
 import wisp.logging.getLogger
-import jakarta.inject.Inject
 import kotlin.reflect.KClass
 
 class AccessInterceptor private constructor(
```

Comment:
it's weird to me that a private constructor ends up in the public api files, maybe a bug with our api dump?